### PR TITLE
PLAT-141606: Input icons not aligned with text

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
`iconBefore` and `iconAfter` are not positioned correctly on Carbon, Cobalt, Copper, Electro, Titanium skins

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added `align-self: center` to icons for all skins

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-141606

### Comments

Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/ArcSlider` style to match latest design for Silicon skin
 - `agate/Button` badge background color for Cobalt and Copper skins
 - `agate/ContextualPopupDecorator` style to match latest design for Silicon skin
+- `agate/Input` to position correctly iconBefore and iconAfter on Carbon, Cobalt, Copper, Electro, Titanium skins
```
Comment (MikyungKim) [inline CHANGELOG.md:17]: I think this PR is too old.. Let's do merge develop.

Comment (MikyungKim) [reply CHANGELOG.md:17]:
```suggestion
- `agate/Input` to position correctly `iconBefore` and `iconAfter` on Carbon, Cobalt, Copper, Electro, and Titanium skins
```

Comment (daniel-stoian-lgp) [reply CHANGELOG.md]: indeed. very old. The merge is done

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/Heading` style to match latest design for Silicon skin
 - `agate/Heading` position of underline when size="tile" on Carbon, Cobalt, Copper, Electro, Titanium skins
 - `agate/Input` to match latest design for Silicon skin
+- `agate/Input` to position correctly `iconBefore` and `iconAfter` on Carbon, Cobalt, Copper, Electro, and Titanium skins
```
Comment (MikyungKim) [inline CHANGELOG.md:28]: You put this to the wrong version :)

Comment (daniel-stoian-lgp) [reply CHANGELOG.md]: ups

**Comment by MikyungKim** [comment]
It's not reproducible on the recent develop branch. Closing this out.

**Pull request closed without merging**